pkg/pipeline/transform/kubernetes/enrich.go

```diff
@@ -21,10 +21,12 @@ func InitFromConfig(config api.NetworkTransformKubeConfig) error {
 	return informers.InitFromConfig(config)
 }
 
-func Enrich(outputEntry config.GenericMap, rule api.K8sRule) {
+func Enrich(outputEntry config.GenericMap, rule *api.K8sRule) {
 	ip := fmt.Sprintf("%s", outputEntry[rule.Input])
-	// TODO: replace this by a new rule input
-	mac := fmt.Sprintf("%s", outputEntry[strings.Replace(rule.Input, "Addr", "Mac", -1)])
+	var mac string
+	if len(rule.MacInput) > 0 {
+		mac = fmt.Sprintf("%s", outputEntry[rule.MacInput])
```

Comment:
For info, for performance we should avoid using `fmt.Sprintf("%s", ...)` whenever possible especially when it's part of the flows processing path - it's not only here but in a bunch of other places too. I've just created a task for that: https://issues.redhat.com/browse/NETOBSERV-1799